tests/__snapshots__/test_help_messages.ambr

```diff
@@ -416,14 +416,26 @@
    defined in the project definition file.                                        
                                                                                   
   ╭─ Options ────────────────────────────────────────────────────────────────────╮
-  │ --force                  When enabled, this option causes the command to     │
-  │                          implicitly approve any prompts that arise. You      │
-  │                          should enable this option if interactive mode is    │
-  │                          not specified and if you want perform potentially   │
-  │                          destructive actions. Defaults to unset.             │
-  │ --project  -p      TEXT  Path where the Snowflake Native App project         │
-  │                          resides. Defaults to current working directory.     │
-  │ --help     -h            Show this message and exit.                         │
+  │ --force                                  When enabled, this option causes    │
+  │                                          the command to implicitly approve   │
+  │                                          any prompts that arise. You should  │
+  │                                          enable this option if interactive   │
+  │                                          mode is not specified and if you    │
+  │                                          want perform potentially            │
+  │                                          destructive actions. Defaults to    │
+  │                                          unset.                              │
+  │ --cascade          --no-cascade          When enabled, will also drop all    │
+  │                                          application objects owned by the    │
+  │                                          application within the account.     │
+  │                                          Defaults to false.                  │
+  │ --interactive  -i                        When enabled, this option displays  │
```

Comment:
When enabled, the command displays...

(out of curiosity, if not terminals, where are the prompts displayed?)

Reply:
When a human runs the CLI we can ask questions interactively as the script executes. When a script runs it, the "answers" should be provided in advance via options.

Reply:
nit: I think Mike's suggestion for "this option displays" -> "this command displays" is a good one

Reply:
Sure, let's align the wording across all commands/options? It's not really tied to this PR